CONTRIBUTING.md

```diff
@@ -6,31 +6,270 @@ We want to make contributing as easy and transparent as possible.
 
 Please read through this document before submitting any contributions to ensure your contribution goes to the correct code repository and we have all the necessary information to effectively respond to your request.
 
+## Development Setup
```

Comment:
can you integrate the development notes from https://github.com/Second-Hand-Friends/kleinanzeigen-bot/blob/main/README.md#development-notes into this file and remove them from there?